syntax = "proto3";

message UserRequest {
    int32 user_id = 1;
}

message UserPreferences {
    int32 id = 1;
    string color = 2;
    string fuel = 3;
    string transmission = 4;
    string max_price = 5;
    string year = 6;
    string manufacter = 7;
}

message UserPreferencesResponse {
    UserPreferences preferences = 1;
}

service Users {
    rpc Preferences (UserRequest) returns (UserPreferencesResponse);
}